statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -146,39 +146,42 @@ func (wr *WidgetsRender) prepareInstanceInfo() {
 	countConsensusAcceptedBlocks := wr.presenter.GetCountConsensusAcceptedBlocks()
 
 	if shardId == uint64(core.MetachainShardId) {
-		consensusInfo = fmt.Sprintf("Count consensus participant: %d | Signed blocks headers: %d", countConsensus, countConsensusAcceptedBlocks)
-
+		consensusInfo = fmt.Sprintf("Validator signed meta blocks: %d | Signed blocks headers: %d", countConsensus, countConsensusAcceptedBlocks)
 	} else {
-		consensusInfo = fmt.Sprintf("Consensus accepted / signed blocks: %d / %d", countConsensusAcceptedBlocks, countConsensus)
+		consensusInfo = fmt.Sprintf("Validator signed blocks %d | Number accepted: %d", countConsensus, countConsensusAcceptedBlocks)
```

Comment:
countConsensusAcceptedBlocks is not incremented as expected for meta. It adds also the shard headers. Please run a local test and analyse what it outputs. I do not think the function saveMetachainCommitBlockMetrics from metrics.go calls correctly the countNotarizedHeaders when the node is a meta node.

Reply:
This is the way it was wrote at the beginning. This is why we have `Signed blocks headers` for a meta node in TermUI.